cli/chaosli/cmd/explain.go

```diff
@@ -7,26 +7,203 @@ package cmd
 
 import (
 	"fmt"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+	"strings"
 
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	chaostypes "github.com/DataDog/chaos-controller/types"
+	"github.com/ghodss/yaml"
 	"github.com/spf13/cobra"
 )
 
+type Explanations struct {
```

Comment:
This is beyond my `go` expertise, sadly, so I don't know which approach is more "idiomatic", but because this struct doesn't actually carry any state, I think you can just turn `separator` into a `const`, and make all of its methods into functions.

Reply:
I was thinking about treating it sort of like a static class and using encapsulation to keep everything in one struct. There could also be future data points and better testing working with the single struct versus importing a whole file. 

What are your thoughts on that, do you think its still weird to have it coded this way?

Reply:
It is not weird to code it this way, it works just fine. I have a soft preference for keeping code as stateless as possible, and given there's no requirement to attach these functions to a struct, I would err on the side of not doing so. While, admittedly, I can foresee possible reasons to need state when explaining a yaml in possible futures, stateless code is "on average" more flexible to test and easier to read than stateful code. So on the principle of "yagni", I'd say it's better to drop the struct for now, but consider this a -0 at most.

Reply:
yeah this makes sense, thanks for explaining. Lets change it.